### ISSUE TITLE
No colors showing on latest Beta

### ISSUE THREAD

**Issue by MattBidewell** [abe33/atom-pigments#310]

**Comment by C-Bouthoorn** [comment]
What OS are you running, how big is the project, what language is that project/file written in? It would be nice if you could share some more information with us

**Comment by MattBidewell** [comment]
Atom Verson 1.18.0-Beta.
Language is css.
Project is a brand new project but also happening on existing projects of varying file sizes.

**Comment by C-Bouthoorn** [comment]
Confirmed on clean Atom `1.18.0-beta`.

### Results
* none of the `marker-type`'s show a color
* `pigments:reload` finds 1 path
* `pigments:find colors` actually finds and displays the color


### Expectations
Expected `#abcdef` to get a slightly blue'ish background color.


### Screenshots
![screenshot from 2017-05-20 17-06-01](https://cloud.githubusercontent.com/assets/10348289/26276790/a5d49b86-3d7e-11e7-9fcb-008882c0f9d6.png)



### Code
```css
html {
  background-color: #abcdef;
}
```


### Versions

Pigments: `0.39.1`
Atom: `1.80.0-beta`
OS: `Linux 4.10.13-1-ARCH x86_64 GNU/Linux`

```
$ atom-beta --version
Atom    : 1.18.0-beta0
Electron: 1.3.15
Chrome  : 52.0.2743.82
Node    : 6.5.0
```

```
$ apm-beta --version
apm  1.18.1
npm  3.10.10
node 6.9.5 x64
python 3.6.1
git 2.13.0
```


### `pigments:report`
```json
{
  "atom": "1.18.0-beta0",
  "pigments": "0.39.1",
  "platform": "linux",
  "config": {
    "traverseIntoSymlinkDirectories": false,
    "sourceNames": [
      "**/*.css",
      "**/*.styl",
      "**/*.stylus",
      "**/*.less",
      "**/*.sass",
      "**/*.scss"
    ],
    "ignoredNames": [
      "vendor/*",
      "node_modules/*",
      "spec/*",
      "test/*"
    ],
    "ignoredBufferNames": [],
    "extendedSearchNames": [
      "**/*.css"
    ],
    "supportedFiletypes": [
      "*"
    ],
    "filetypesForColorWords": [
      "css",
      "less",
      "styl",
      "stylus",
      "sass",
      "scss"
    ],
    "ignoredScopes": [],
    "autocompleteScopes": [
      ".source.css",
      ".source.css.less",
      ".source.sass",
      ".source.css.scss",
      ".source.stylus"
    ],
    "extendAutocompleteToVariables": false,
    "extendAutocompleteToColorValue": false,
    "autocompleteSuggestionsFromValue": false,
    "maxDecorationsInGutter": 100,
    "markerType": "native-background",
    "sassShadeAndTintImplementation": "compass",
    "sortPaletteColors": "none",
    "groupPaletteColors": "none",
    "mergeColorDuplicates": false,
    "delayBeforeScan": 500,
    "ignoreVcsIgnoredPaths": true,
    "notifyReloads": true,
    "dismissableReloadNotifications": true
  },
  "project": {
    "config": {},
    "paths": [
      "<root>/test.css"
    ],
    "variables": {
      "colors": 0,
      "total": 0
    }
  }
}
```

**Comment by gusbemacbe** [comment]
It also happens in macOS, on a LESS file.

**Comment by pingshunhuangalex** [comment]
This bug is fixed in `Atom 1.18.0-beta1` now.
